src/concepts/pipeline/glean_data.md

```diff
@@ -21,19 +22,8 @@ The top-level groups are:
 
 ### Built-in metrics
 
-Accessing columns from the `client_info`, and `ping_info` group is reasonably straightforward.
-
-For example, to access the `client_id` of a ping, use the column `client_info.client_id`.
-
-```sql
--- Count unique Client IDs observed on a given day
-SELECT
-  count(distinct client_info.client_id)
-FROM
-  org_mozilla_fenix.baseline
-WHERE
-  date(submission_timestamp) = '2019-11-11'
-```
+[Core attributes sent with every ping](https://mozilla.github.io/glean/book/user/pings/index.html#glean-pings) are mapped to the `client_info` and `ping_info` sections. For example,
+the client id is mapped to a column called `client_info.client_id`.
```

Comment:
May be useful to link directly to the glean documentation for these:

https://mozilla.github.io/glean/book/user/pings/index.html#the-ping_info-section
https://mozilla.github.io/glean/book/user/pings/index.html#the-client_info-section